src/components/ErrorBoundary/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import ErrorBoundary from './ErrorBoundary';
+
+describe('Error Boundary', () => {
+  test('Error Boundary', () => {
+    const ThrowError = () => {
+      throw new Error('Test');
+    };
+
+    render(
+      <ErrorBoundary>
+        <ThrowError />
+      </ErrorBoundary>
+    );
+
+    expect(screen.getByTestId('error-boundary')).toBeVisible();
```

Comment:
Try to avoid using data-testid for testing purposes. Please use another methods.